backend/courses/models.py

```diff
@@ -1396,16 +1402,14 @@ def save(self, *args, **kwargs):
         does not throw an error.
         It then calls the normal save method.
         """
-        if self.phone is not None and self.phone.strip() == "":
+        if not self.phone or not self.phone.strip():
             self.phone = None
-        if self.phone is not None:
-            try:
-                phone_number = phonenumbers.parse(self.phone, "US")
-                self.phone = phonenumbers.format_number(
-                    phone_number, phonenumbers.PhoneNumberFormat.E164
-                )
-            except phonenumbers.phonenumberutil.NumberParseException:
-                raise ValidationError("Invalid phone number (this should have been caught already)")
+        if self.phone:
+            # self.phone validated by `validate_phone`
```

Comment:
Nit but I think this should be 'should be validated' since technically it's not guaranteed the validator is called (https://docs.djangoproject.com/en/4.2/ref/models/instances/#django.db.models.Model.clean_fields)